__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const app = require('../app');
+const request = require('supertest')
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index.js');
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe('/api/topics', () => {
+    test('Request to an invalid endpoint receives a 404 error', () => {
+        return request(app)
+        .get('/api/forklift')
+        .expect(404)
+    })
+    test('GET:200 sends an array of topic objects to the client', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            console.log(response.body.topics)
+            expect(response.body.topics.length).toBe(3)
+            response.body.topics.forEach((topic) => {
+                expect(typeof topic.slug).toBe('string')
+                expect(typeof topic.description).toBe('string')
+            })
```

Comment:
Excellent to see a length check before the forEach, just incase we get an empty array this test will correctly fail